pollination_handlers/outputs/daylight.py

```diff
@@ -105,3 +106,41 @@ def ill_credit_json_from_path(eui_json):
     for key in sorted(data.keys()):
         results.append('{}: {}'.format(key, data[key]))
     return results
+
+
+def read_leed_datacollection_from_folder(result_folder):
+    """Read LEED Daylight Option I datacollections """
+    # check that the required files are present
+    if not os.path.isdir(result_folder):
+        raise ValueError('Invalid result folder: %s' % result_folder)
+    grid_json = os.path.join(result_folder, 'grids_info.json')
+    if not os.path.isfile(grid_json):
+        raise ValueError('Result folder contains no grids_info.json.')
+
+    # load the list of grids and gather all of the results
+    with open(grid_json) as json_file:
+        grid_list = json.load(json_file)
+    results = []
+    for grid in grid_list:
+        grid_id = grid['full_id']
+        result_file = os.path.join(result_folder, '{}.{}'.format(grid_id, 'json'))
+        with open(result_file) as json_file:
+            data = json.load(json_file)
+            datacollection = HourlyContinuousCollection.from_dict(data)
```

Comment:
I don't know how this fits into your larger workflow but I just wanted to make sure you knew that there's a dedicated module for serializing DataCollections to/from different file formats:

https://www.ladybug.tools/ladybug/docs/ladybug.datautil.html

Serializing to/from CSV tends to be the most space efficient in terms of file size and it's also the one that most users seem to prefer for opening the data in other programs. So I might recommend this over the JSON format for DataCollections that you are using here.

Reply:
Thanks, Chris! I see. I think what I did in the PR fits better for my workflow, but I will keep this in mind in the future.